src/transaction/invoke_function.rs

```diff
@@ -27,6 +27,7 @@ use num_traits::Zero;
 
 use super::Transaction;
 
+/// Represents an InvokeFunction transaction in the starknet network.
```

Comment:
I think we should mention it is called `INVOKE_FUNCTION` in the RPC API and in starkscan.